src/client/components/Swipe/Swipe.jsx

```diff
@@ -216,68 +215,30 @@ class Swipe extends Component {
 
     return (
       <div className="swipe">
-        <div className="swipe-header-primary">
-          <div className="city">
-            {city}
-          </div>
-        </div>
-        <div className="swipe-header-secondary">
-          <BackButton
-            onClick={() => this.routeChange(PATHS.trips())}
-            // onClick={() => this.routeChange(PATHS.trips(userId))}
-          />
-          {places.length > 0 && (
-            <div>
-              {this.renderList(this.state.swipeList)}
+        <div className="swipe-header">
+          <div className="swipe-header-primary">
+            <div className="city">
+              {city}
             </div>
-          )}
-          <img
-            className="icon-list"
-            src="/assets/common/icon-list.png"
-            onClick={() => this.routeChange(PATHS.list(tripId))}
-            // onClick={() => this.routeChange(PATHS.list(userId, tripId))}
-          />
+          </div>
+          <div className="swipe-header-secondary">
+            <HomeButton
+              onClick={() => this.routeChange(PATHS.trips())}
+            />
+            {places.length > 0 && (
+              <div>
+                {this.renderList(this.state.swipeList)}
+              </div>
```

Comment:
don't render a div outside of a function, put it within the render function and give it a `className`

so...

```js
{places.length > 0 && this.renderList(this.state.swipeList)}
```

```js
renderList(swipeList) {
   ...
   return (
   <div className="toggle-button-group">
      <ToggleButtonGroup className="toggle-buttons" name="toggle-button" value={swipeList} onChange={this.listChange}>
          <ToggleButton className="toggle-button-selected" name="food" value={1}>food</ToggleButton>
          <ToggleButton className="toggle-button-unselected" name="places" value={2}>places</ToggleButton>
       </ToggleButtonGroup>
   </div>
}
```